### PR TITLE
LIG-673: ApiWorkflowClient can receive LIGHTLY_TOKEN env variable

### PR DESCRIPTION
**Pull request by shaundaley39**
- ApiWorkflowClient can be initialized with or without a token argument (alternative: LIGHTLY_TOKEN can be provided)

### CODE REVIEW

File: lightly/api/api_workflow_client.py [hunk 1]
```diff
@@ -73,12 +73,14 @@ class ApiWorkflowClient(_UploadEmbeddingsMixin,
...
 
-    def __init__(self, token: str, dataset_id: str = None, embedding_id: str = None):
+    def __init__(self, token: str = None, dataset_id: str = None, embedding_id: str = None):
 
         self.check_version_compatibility()
 
         configuration = Configuration()
         configuration.host = getenv('LIGHTLY_SERVER_LOCATION', 'https://api.lightly.ai')
+        if token is None:
+            token = getenv('LIGHTLY_TOKEN', None)
```
Comment (IgorSusmelj) [inline lightly/api/api_workflow_client.py:83]: What happens if there is no env variable? Maybe throw a `ValueError` then?

**Review by IgorSusmelj — CHANGES_REQUESTED** [review]
Looks good. But let's make sure there exists an env variable and that we cover the case if there is none.

**Review by MalteEbner — COMMENTED** [review]
And please add a unittest for creating an ApiWorkflowClient() without a token, but with an env variable.

File: lightly/api/api_workflow_client.py (hunk 1 above)
Comment (shaundaley39) [reply lightly/api/api_workflow_client.py:83]: Sure, thrown

**Comment by shaundaley39** [comment]
@MalteEbner unit tests added

**Review by MalteEbner — APPROVED** [review]
This is great! And you see how the code coverage of your newly added lines is 100%.

File: lightly/api/api_workflow_client.py [hunk 2]
```diff
@@ -73,12 +73,18 @@ class ApiWorkflowClient(_UploadEmbeddingsMixin,
...
+            if token is None:
+                raise ValueError(f"Either provide a 'token' argument or export"
```
Comment (IgorSusmelj) [inline lightly/api/api_workflow_client.py:85]: FYI. There is no need to use an f-string in python if you don't include variables. 

https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python

**Review by IgorSusmelj — APPROVED** [review]

**Pull request merged**